extensions/wikia/SoundCloudTag/SoundCloudTagController.class.php

```diff
@@ -38,15 +38,18 @@ public function __construct() {
 		$this->helper = new WikiaIFrameTagBuilderHelper();
```

Comment:
That's true and makes me sad :(

But let's not focus on emotions! ;) The only potential benefit is that the trick with `onParserFirstCallInit` and `onParserAfterTidy` hooks make us sure the Parser won't put any magical, empty paragraphs within tag's output.

Do you want me to remove inheritance from `SoundCloudTagController`?